src/views/pages/name/scottish-limited-partnership-name.njk

```diff
@@ -0,0 +1,150 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      {% include "includes/csrf_token.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ i18n.scottishNamePage.title }}</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            {{ i18n.scottishNamePage.whatIsName }}
+          </label>
+        </h1>
+
+        {{ govukInput({
+          classes: "govuk-input--width-40",
+          errorMessage: errors.scottish_partnership_name if errors,
+          id: "scottishPartnership_name",
+          name: "scottishPartnership_name",
+          value: scottishPartnership_name,
+          fieldset: {
+            legend: {
+              isPageHeading: false,
+              classes: "govuk-fieldset__legend--xl"
+            }
+          },
+          hint: {
+            text: i18n.scottishNamePage.whatIsNameHint
+          },
+          attributes : {
+            required : true,
+            "aria-describedby": "partnership_name-hint",
+            minLength: "3"
+          },
+          suffix: {
+              text: "Limited Partnership"
+            }
+          }) }}
+      </div>   
+
+      {{ govukInsetText({
+        html: '<p>' + i18n.namePage.existingPartnershipHint + '</p><p>' + i18n.namePage.sensitiveWordHint + '</p><p>' + i18n.namePage.existingTrademarkHint + '</p>'
```

Comment:
the i18n values in here should use scottishNamePage instead of namePage

Reply:
i18n now using scottishNamePage instead of namePage